hack/.linted_packages

```diff
@@ -0,0 +1,506 @@
+cluster/addons/fluentd-elasticsearch/es-image
```

Comment:
`hack/.linted_packages` has been removed in https://github.com/kubernetes/kubernetes/pull/47377. `.golint_failures` replaces it.